demo/runtime/src/lib.rs

```diff
@@ -33,48 +36,12 @@ extern crate demo_primitives;
 
 extern crate integer_sqrt;
 
+#[macro_use] pub mod dispatch;
```

Comment:
the macros from `dispatch` aren't used outside of it as far as I can see

Reply:
`impl_dispatch` is.